File: Question2.java
class Question2 {

	public static void main(String[] args) {
		// TODO 自動生成されたメソッド・スタブ
		for (int i=1; i<=100; i++) {
			int num = fibo(i);

			if (num > 1000) {
				break;
			}
			System.out.printf("%3d ", num);
		}
		System.out.println();
	}

	static int fibo (int num) {
		if (num == 1 || num == 2) {
			return 1;
		} else {
			return fibo (num - 1) + fibo (num - 2);
		}
	}
}

File: Question2.cs
using System;
namespace Algorithm_Practice {
    class Question2 {
        public Question2() {
        }

        public void Action(int n) {
            Console.WriteLine("Fibonacci: " + n + "// Fibo(n) < 1,000");
            for (int i = 1; i <= n; i++) {
                int fibo = Fibo(i);

                if (fibo > 1000) {
                    break;
                }
                Console.Write("{0:d} ", fibo);
            }
            Console.WriteLine();
        }

        private int Fibo(int n) {
            if (n == 1 || n == 2) {
                return 1;
            } else {
                return Fibo(n - 1) + Fibo(n - 2);
            }
        }
    }
}
